### ISSUE TITLE
Template is not set correctly

### ISSUE THREAD

**Issue by whollaus** [getgrav/grav-plugin-admin#1573]
[Latest Admin Version: Grav v1.6.0-beta.7 - Admin v1.9.0-beta.7 - Quark v2.0.0-beta.4]

Add an new page (Blog Example; but also default...):
![grafik](https://user-images.githubusercontent.com/4582757/50635432-69e71e80-0f52-11e9-94a8-fc06001fc9cb.png)

On the next page, the template is not set correctly (See Page Template Field). All modular blueprints are listed as a selection
![grafik](https://user-images.githubusercontent.com/4582757/50635460-91d68200-0f52-11e9-9c93-8ba26744e1fa.png)

Thank you very much.

**Comment by whollaus** [comment]
Another example with Item template:
![grafik](https://user-images.githubusercontent.com/4582757/50636316-e5969a80-0f55-11e9-8ad6-f4d60b3bd875.png)
![grafik](https://user-images.githubusercontent.com/4582757/50636343-0232d280-0f56-11e9-87b3-81bf24e74229.png)
After first save the Blog Template was selected in the form:
![grafik](https://user-images.githubusercontent.com/4582757/50636379-25f61880-0f56-11e9-8bfb-985087b7a6e3.png)
So, if an user save again, the blog template (with blog config) will activated:
![grafik](https://user-images.githubusercontent.com/4582757/50636458-68b7f080-0f56-11e9-9711-096476147ca6.png)

**Comment by paulhibbitts** [comment]
Hi @rhuk, I've just noticed this issue (at least I think it is the same issue), but in my case Blueprints that are extended are not being picked up in the Admin New Page dialog 1.9 Beta 7.

This blueprint is being picked up in Grav 1.5.x, but not 1.6x Beta 7:

```
title: Embedly Card Options
'@extends':
    type: default
    context: blueprints://pages
form:
  fields:
    tabs:
      type: tabs
      active: 1
      fields:
        content:
            fields:
                header.title:
                  replace@: true
                  type: text
                  size: long
                  label: Page Title

                header.embedly_card_title:
                    ordering@: 2
                    type: text
                    size: long
                    label: Embedly Card Title

                header.display_embedly_card_title:
                    ordering@: 3
                    type: toggle
                    label: Display Embedly Card Title
                    default: 0
                    highlight: 1
                    options:
                        1:  Yes
                        0:  No
                    validate:
                        type: bool

                header.embedly_card_alignment:
                    ordering@: 4
                    type: select
                    size: small
                    label: Embedly Card Alignment
                    default: left
                    options:
                        left: Left
                        center: Center
                        right: Right

                header.embedly_card_url:
                    ordering@: 5
                    type: text
                    size: long
                    label: Embedly Card Web Page URL
                    validate:
                      type: URL

        options:
          fields:
            publishing:
              fields:
                header.published:
                  default: 1

        advanced:
          fields:
            overrides:
              fields:
                header.visible:
                  default: 1
```

Please let me know if more info is helpful etc.

Thanks very much,
Paul

**Comment by rhukster** [comment]
Fixed in 1.6
